src/main/java/org/jfrog/hudson/pipeline/common/executors/MavenExecutor.java

```diff
@@ -69,7 +69,7 @@ public void execute() throws Exception {
         maven3Builder.perform(build, launcher, listener, extendedEnv, ws, tempDir);
         String generatedBuildPath = extendedEnv.get(BuildInfoFields.GENERATED_BUILD_INFO);
         buildInfo.append(Utils.getGeneratedBuildInfo(build, listener, launcher, generatedBuildPath));
-        buildInfo.appendDeployableArtifacts(extendedEnv.get(BuildInfoFields.DEPLOYABLE_ARTIFACTS), tempDir, listener);
+        buildInfo.appendDeployableArtifactsByModule(extendedEnv.get(BuildInfoFields.DEPLOYABLE_ARTIFACTS), tempDir, listener);
```

Comment:
Let's add the following comment for this line (I copied it from the **GradleExecutor** class):
// Read the deployable artifacts map from the 'json' file in the agent and append them to the buildInfo object.